doc/source/schema/PyMC3_schema_example.ipynb

```diff
@@ -327,7 +327,10 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this example, each variable has as dimension a combination of the following 3: `chain`, `draw` and `developer`. Moreover, each dimension has specific coordinate values. In the case of `chain` and `draw` it is an integer identifier starting at `0`; in the case of `developer` dimension, its coordinate values are the following strings: `[\"Alice\", \"Bob\", \"Cole\", \"Danielle\", \"Erika\"]`."
+    "In this example, each variable for its dimension has a combination of the following three: `chain`, `draw` and `developer`. Moreover, each dimension has specific coordinate values.\n",
```

Comment:
> In this example, each variable for its dimension has a combination of the following three: `chain`, `draw` and `developer`.

I am not sure this is an improvement over the old wording, I think this can now be more confusing now and doesn't really follow the terminology defined in the schema. "dimension" should probably be plural (either in this new wording or the original one). This alone might be enough, I am sometimes overwary of constructions that differ too much from catalan or spanish ones.